docs/alerts/monitors/use-playbooks-with-monitors.md

```diff
@@ -0,0 +1,144 @@
+---
+id: use-playbooks-with-monitors
+title: Use Automated Playbooks with Monitors
+description: Learn how to use Automation Service playbooks with monitors.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This article describes how to use automated playbooks with monitors. A [playbook](/docs/platform-services/automation-service/automation-service-playbooks/) is a predefined set of actions and conditional statements that run in an automated workflow to respond to a certain event or incident type. For example, suppose that a monitor detects suspicious behavior that could indicate a security problem. When the monitor sends the alert, it could also run an automated playbook to respond to the event. 
+
+To add a playbook to a monitor, see [Add an automated playbook to a monitor](#add-an-automated-playbook-to-a-monitor) below.
+
+To create a monitor to detect suspicious behavior (anomalies), see [Create a "smart alert"](#create-a-smart-alert) below.
+
+## Prerequisites
+
+Automated playbooks are provided by the [Automation Service](/docs/platform-services/automation-service). To use automated playbooks with monitors, you must perform tasks to enable the Automation Service. See the [prerequisites for the Automation Service](/docs/platform-services/automation-service/about-automation-service/#prerequisites).
+
+## Access playbooks for monitors
+
+When working with monitors, use the following ways to access playbooks in the Automation Service.
+
+### When creating a monitor
+
+1. [Create a monitor](/docs/alerts/monitors/create-monitor).
+1. Go to the **Monitor Details** section.
+1. Beneath the **Playbook** field, click the **Manage Playbooks** button. <br/>Playbooks are displayed in the Automation Service.
+
+For more information, see [Add an automated playbook to a monitor](/docs/alerts/monitors/use-playbooks-with-monitors#add-an-automated-playbook-to-a-monitor) below.
+
+### When viewing an alert
+
+1. On the [alert page](/docs/alerts/monitors/alert-response/#alert-page), open an alert that has attached playbooks.
+1. Click the **Playbook** button. The attached playbooks are displayed.
+1. Click the name of an attached playbook. <br/>The playbook is opened in the Automation Service.
+
+For more information, see [View playbooks for an alert](/docs/alerts/monitors/use-playbooks-with-monitors#view-playbooks-for-an-alert) below.
+
+### From the main menu
+
+If you have Cloud SIEM installed, you can also access playbooks from the main menu.
```

Comment:
I'd add this in case a customer lands on this page not knowing what Cloud SIEM is

```suggestion
If you have [Cloud SIEM](/docs/cse) installed, you can also access playbooks from the main menu.
```